### PR TITLE
Starting Location fixes for Special Assignments.

### PR DESCRIPTION
**Pull request by dribbleondo**
My bad. I mean, I dislike these discount-permanent Elusive Targets, hence my week-long absence.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -68,6 +68,14 @@ <h3 style="display:inline">Changelog</h3>
         Show
       </button>
       <ul id="changelog" style="display:none">
+        <li class="versionnumber">0.532.5</li>
```
Comment (TheKotti) [inline about.html]: Remove this whole thing

File: about.html [hunk 2]
```diff
@@ -68,6 +68,14 @@ <h3 style="display:inline">Changelog</h3>
...
+        <li>
+          Fixed this changelog to accurately reflect how many locations were actually removed (four, not two).
+        </li>
+        <li class="versionnumber">0.532</li>
+        <li>
+          Fixed four incorrect starting locations in the Permanant Elusi- er.."Special Assignments" (If you're curious, it was the "Shaman's Hut" and "Village Bus Stop" starting points in Columbia, and the "Chawls" and "Photoshoot" starting points in Mumbai)
```
Comment (TheKotti) [inline about.html]: Just call them special assignments, no need to be a smartass here.

**Review by TheKotti — CHANGES_REQUESTED** [review]
Would strongly recommend looking into rebasing so there's not 4 commits over a couple of json entries.

File: about.html (hunk 1 above)
Comment (dribbleondo) [reply about.html]: done.

File: about.html (hunk 2 above)
Comment (dribbleondo) [reply about.html]: doneso.

**Comment by dribbleondo** [comment]
I've tried rebasing it, and Github won't update the pull requests as such. I'm just gonna close this commit, delete the branch I have, re-fork your branch, then add the commits that way. A very annoying solution.

I'm probably doing something wrong.

**Pull request closed without merging**